_partials/gcp_setup_mid.md

```diff
@@ -0,0 +1,6 @@
+- Store the service account json file somewhere you'll remember, for example `/Users/YOUR_USER_NAME/Documents/gcp/SERVICE_ACCOUNT_JSON_FILE_CONTAINING_YOUR_SECRET_KEY.json`
```

Comment:
Wouldn't it be better to create a separate instruction for Linux? We don't have `/Users` as absolute path, but `/home/` - less confusion for students on the first day :)

Reply:
I'll do that 👌

Reply:
@zzuziak just added equivalent storage locations for mac / linux under ~/code/github_nickname/gcp/